homeassistant/components/zwave_js/entity.py

```diff
@@ -92,7 +92,13 @@ def unique_id(self) -> str:
     @property
     def available(self) -> bool:
         """Return entity availability."""
-        return self.client.connected and bool(self.info.node.ready)
+        return (
+            self.client.connected
+            and bool(self.info.node.ready)
```

Comment:
Why we cast to bool

Reply:
I think to satisfy mypy as the value is optional